### ISSUE TITLE
Training chart went to loss = 0.0000 when training after about a half hour

### ISSUE THREAD

**Issue by cylemmulo-1** [AlexeyAB/darknet#6977]
![yolotrain](https://user-images.githubusercontent.com/34980486/99156644-21d5ce80-2688-11eb-8deb-830d8b7ffb88.png)
![yolotrain2](https://user-images.githubusercontent.com/34980486/99156658-4336ba80-2688-11eb-934b-85c3c7a7f70c.png)

I started training a custom dataset for the first time. Running this on Windows 10. Everything looked okay my command is:

./darknet.exe detector train data/obj.data cfg/yolov3-customkyle.cfg darknet53.conv.74

It looked normal sitting at like loss of 1000+ like I see people having, but then as you see it went like a rollercoaster and shot down to just a loss of 0. As far as I can tell it's not supposed to do that. There would always be some loss.

Would this be it running at a certain point to a bad set of training inputs or could anyone maybe point me some directions to look at.

**Comment by stephanecharette** [comment]
Out of curiosity, what were the results like?  Did the weights file work, or did it give garbage?

**Comment by cylemmulo-1** [comment]
Well I used OIDv4 to download an imageset. Testing it the same way I saw in a guide and it looked legit. I have yet to test it out though, it just finished a little bit ago. I will report back when I try it

**Comment by cylemmulo-1** [comment]
Just piping in again, I did a small amount of testing with some images and it doesn't appear to be doing anything. So a bit inconclusive atm.

**Comment by RashikShahjahan** [comment]
@cylemmulo-1 , did you fix it? I am having a similar issue.

**Comment by cylemmulo-1** [comment]
No sadly I had two computers that did this. I'm sure I had something setup wrong. But I haven't trained any datasets since this. If you find out anything I'd love to hear.
